test/bugs/ui-7618.js

```diff
@@ -0,0 +1,217 @@
+// https://github.com/swagger-api/swagger-ui/issues/7618
```

Comment:
We've broken away from this pattern of naming tests some time ago. Please incorporate your tests into appropriate context, in this case https://github.com/lipnitsk/swagger-js/blob/e5788a8d1233b6cbbc0821d43392393c5a6ec607/test/specmap/lib.js. By incorporating the test into proper context you won't need this big fixture anymore.

Reply:
incorporated test by extending a similar test case within `test/specmap/all-of.js`